Dockerfile

```diff
@@ -5,7 +5,7 @@ COPY . /main_ws/src/
 # this:
 # 1) builds the application
 # 2) packages the application as .deb & writes it to build_output/
-RUN /packaging/build-and-package-as-deb.sh -o build_output/
+RUN /packaging/build.sh
```

Comment:
This didn't work before. If I recall correctly, this was the only repo that needed a different build process than the `build.sh`. I'm glad that you fixed it! But I'm left wondering why it works now and didn't before?

I'm also wondering how this works because I think `build.sh` writes the .deb to `/main_ws/something.deb` and the runtime image expects the artifact to be at `/main_ws/src/build_output/...`?

https://github.com/tiiuae/rplidar_ros2/blob/2329730830da3dbf89deaa4ade13d87170313bb4/Dockerfile#L19

Reply:
If you compare this to mocap_pose, build.sh writes the .deb to where I mentioned above, so I'm scratching my head why the build succeeded :thinking: 

https://github.com/tiiuae/mocap_pose/blob/2731c410fd72e2c4b4a8984b6e123346211bea5d/Dockerfile#L22

Reply:
Actually, I had to change the build script used because after the fakeroot package was removed the build-and-package-as-deb.sh script got broken.
And you are correct about the path. I will take a look in more detail.

Reply:
I don't understand how it was working when it obviously was using the wrong path to a file which does not exist. The installed debian package had the same hash a the one built. 
In addition to this, the same bug was in depthai_ctrl and it works there too.

Reply:
Fixed.